data/tlg0031/tlg001/tlg0031.tlg001.perseus-eng2.xml

```diff
@@ -78,6 +80,7 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?>
         </profileDesc>
         
         <revisionDesc>
+            <change when="2024-03-15" who="Lisa Cerrato">Minor header changes and updating</change>
```

Comment:
thanks for the credit! ;)

Reply:
Your welcome, now if I can just remember to do it more than once.

Reply:
Actually, I meant that your name should be here, not mine.

Reply:
I thought this was supposed to be your update in 2025, which I don't see credited.

Reply:
Sigh. Wrong people and wrong years, another day in the life of metadata meltdown! :)